sticky_notes.py

```diff
@@ -81,3 +81,37 @@ def list_stickynotes(self, event):
                 self.delay_task(index,
                                 self.sender(format_note(index, note), event))
             return first_note
+
+    @cmd
+    def del_note(self, event):
+        args = event.args
+        from_nick = event.meta['nick']
+
+        if len(args) < 1:
+            return "Missing nick argument."
+
+        if len(args) < 2:
+            return "Missing note number argument."
+
+        if len(args) >= 2:
+            nick = args[0]
+            note_num = args[1]
+
+            if nick in self.notes:
+                notes = self.notes[nick]
+                print len(self.notes[nick])
+                to_pop = -2
```

Comment:
@JakubNvk personally, I think you do not need this variable at all. you can simply set `note_num` to -1 if you detect that `note_num == "last"` and then `note_num` should become your `to_pop`.